plugins/kubernetes/test/test_helper.rb

```diff
@@ -106,7 +106,11 @@ def kubernetes_fake_raw_template
     Kubernetes::ReleaseDoc.any_instance.stubs(raw_template: read_kubernetes_sample_file('kubernetes_deployment.yml'))
   end
 
+  def sample_file_path(file_name)
```

Comment:
this is a global method so it needs to be namespaced to kubernetes ... kubernetes_sample_file_path